package.json

```diff
@@ -103,7 +103,7 @@
     "html-loader": "^0.5.1",
     "html-webpack-plugin": "^2.28.0",
     "image-webpack-loader": "^3.3.1",
-    "imagemin-pngquant": "^9.0.2",
+    "imagemin-pngquant": "5.0.1",
```

Comment:
why did we need to downgrade?

Reply:
I see why you needed the package installed. 
https://github.com/imagemin/imagemin-pngquant/issues/46

Does imagemin-pngquant have an alternative that does the same job but less buggy?

Reply:
need to spend time to go through this if we want to replace it.
currently either the testing framework or the dependencies are super fragile in circleci pipeline

better look at moving from circleci to GH actions first

Reply:
we are using manual deployment atm so all these stuffs are hidden under a blanket